src/i18n/de.ts

```diff
@@ -0,0 +1,685 @@
+    "error.annotation.term.unsupportedLanguage": "Der Textanalysedienst unterstützt die Sprache der Definition dieses Begriffs nicht.",
+    "history.label": "Verlauf ändern",
+    "history.loading": "Verlauf wird geladen...",
+    "history.empty": "Der aufgezeichnete Verlauf dieses Vermögenswerts ist leer.",
+    "history.whenwho": "Herkunft",
+    "history.type": "Typ",
+    "history.type.persist": "Erzeugung",
+    "history.type.update": "Aktualisierung",
+    "history.type.delete": "Streichung",
+    "history.changedAttribute": "Attribut",
+    "history.originalValue": "Ursprünglicher Wert",
+    "history.newValue": "Neuer Wert",
+    "changefrequency.label": "Aktivität",
+    "tooltip.copy-iri": "IRI kopieren",
+    "tooltip.copied": "Kopiert!",
+    "table.filter.text.placeholder": "Filtern Sie {count} Datensätze...",
+    "table.filter.select.all": "Alle",
+    "table.sort.tooltip": "Sortieren",
+    "table.paging.first.tooltip": "Gehen Sie zur ersten Seite",
+    "table.paging.previous.tooltip": "Gehen Sie zur vorherigen Seite",
+    "table.paging.next.tooltip": "Gehen Sie zur nächsten Seite",
+    "table.paging.last.tooltip": "Gehen Sie zur letzten Seite",
+    "table.paging.pageSize.select": "Zeigt {pageSize} Elemente pro Seite an",
+    "table.paging.pageSize.select.all": "Alle anzeigen",
+    "stringlistedit.button.add": "Hinzufügen",
+    "stringlistedit.button.add.tooltip": "Klicken Sie, um den eingegebenen Wert zur Liste hinzuzufügen",
+    "stringlistedit.button.remove": "Entfernen",
+    "stringlistedit.button.remove.tooltip": "Wert entfernen",
+    "public.nav.user": "Benutzer nicht angemeldet. Klicken Sie hier, um sich anzumelden.",
+    "public.dashboard.title": "Willkommen bei TermIt!",
+    "public.dashboard.intro": "TermIt ist ein benutzerfreundlicher Vokabularmanager und Terminologieeditor.",
+    "public.dashboard.actions.login": "Einloggen",
+    "public.dashboard.actions.register": "Benutzerkonto erstellen",
+    "public.dashboard.actions.vocabularies": "Durchsuchen Sie Vokabulare und Begriffe",
+    "comments.title": "Kommentare",
+    "comments.create.placeholder": "Geben Sie Ihren Kommentar ein...",
+    "comments.create.submit.title": "Absenden",
+    "comments.list.empty": "Noch keine Kommentare.",
+    "comments.comment.like": "Ich stimme diesem Kommentar zu",
+    "comments.comment.like.on": "Zustimmung entfernen",
+    "comments.comment.dislike": "Ich lehne diesen Kommentar ab",
+    "comments.comment.dislike.on": "Ablehnung entfernen",
+    "comments.comment.edited": "Bearbeitet",
+    "validation.message.tooltip": "Dies ist ein Ergebnis der Qualitätsüberprüfung. ",
+    "snapshots.title": "Snapshot-Verlauf",
+    "snapshots.created": "Snapshot erstellt",
+    "snapshots.show": "Snapshot anzeigen",
+    "snapshots.empty": "Keine vorherigen Snapshots gefunden.",
+    "snapshot.message": "Dieser {Typ} ist ein Snapshot und schreibgeschützt. ",
```

Comment:
Please make sure that labels in `{}` are the same across languages, they are interpolated with actual values when displaying in the UI. In this particular case, it should be `{type}`.